images/bootstrap/create_bazel_cache_rcs.sh

```diff
@@ -70,9 +76,17 @@ make_bazel_rc () {
     # point bazel at our http cache ...
     # NOTE our caches are versioned by all path segments up until the last two
     # IE PUT /foo/bar/baz/cas/asdf -> is in cache "/foo/bar/baz"
-    local cache_id="$(get_workspace),$(hash_toolchains)"
-    local cache_url="http://${CACHE_HOST}:${CACHE_PORT}/${cache_id}"
+    local cache_id
+    cache_id="$(get_workspace),$(hash_toolchains)"
+    local cache_url
+    cache_url="http://${CACHE_HOST}:${CACHE_PORT}/${cache_id}"
     echo "build --remote_http_cache=${cache_url}"
+    # specifically for bazel 0.15.0 we want to set this flag
+    # our docker image now sets BAZEL_VERSION with the bazel version as installed
+    # https://github.com/bazelbuild/bazel/issues/5047#issuecomment-401295174
+    if [[ -z "${BAZEL_VERSION+x}" && "${BAZEL_VERSION}" -eq "0.15.0" ]]; then
```

Comment:
shouldn't the first check be `-n "${BAZEL_VERSION+}"`?

right now I think it'll always be false.

Reply:
though I don't think you need it at all, really. just do something like
```bash
if [[ "${BAZEL_VERSION:-}" -eq "0.15.0" ]]; then
...
```

Reply:
yeah, shellcheck can't fix my misuse of `-z` / `-n` / `-d` :/
fixing

Reply:
so actually this was right but your suggestion is better. "${BAZEL_VERSION+x}" is zero if it evaluates to x and nonzero if bazel_version is actually set.

Reply:
what we really want is:
`if [[ "${BAZEL_VERSION:-}" ="0.15.0" ]]; then`